packages/block-library/src/cover/edit/inspector-controls.js

```diff
@@ -211,9 +211,14 @@ export default function CoverInspectorControls( {
 								}
 								help={
 									<>
-										<ExternalLink href={
-												__('https://www.w3.org/WAI/tutorials/images/decision-tree/')
-												}>
+										<ExternalLink
+											href={
+												// translators: Localized tutorial, if one exists. Link includes list of existing translations.
+												__(
```

Comment:
This is the comment from an earlier version of the patch; we should name the source in the comment, e.g.

`// translators: Localized tutorial, if one exists. W3C Web Accessibility Initiative link has list of existing translations. `

Reply:
@joedolson  Thanks for the feedback, I have implemented the suggestions, please check.